sdk/identity/identity/src/credentials/clientCertificateCredential.ts

```diff
@@ -79,6 +81,9 @@ export class ClientCertificateCredential implements TokenCredential {
       .toUpperCase();
 
     this.certificateX5t = Buffer.from(this.certificateThumbprint, "hex").toString("base64");
+    if (options && options.includeX5c) {
+      this.certificateX5c = [publicKey];
```

Comment:
I may be reading this incorrectly, but it looks like `publicKey` will be only the first capture from the regex. In a well-formed file that would be the signing cert, which may be followed by an arbitrary number of intermediate certs; the x5c header must include all of these.

Reply:
I've brought this PR up to date here: https://github.com/Azure/azure-sdk-for-js/pull/10808

I'll look into the intermediate certs also